app/src/main/res/layout/list_item.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView
```

Comment:
There already is something like this in the Android framework, I think. Something like `aandroid.R.layout.simple_list_item`?

Reply:
fixed